### PR TITLE
DDF-1866: Updated OSGi bundles documentation.

### PR DESCRIPTION
**Pull request by ricklarsen**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/539" target="_blank">ReviewNinja

**Comment by ricklarsen** [comment]
Documentation only update. Bamboo error incorrect.

File: distribution/docs/src/main/resources/_contents/integrating-contents.adoc [hunk 1]
```diff
@@ -48,23 +52,26 @@ Metadata that matches any of the datatypes (and associated versions if specified
...
 For example, a Contextual and Spatial search could be combined into one Compound search to search for certain text in metadata in a particular region of the world.
 
 ==== Search Interfaces
 
-===== {branding} Search UI Application
-The {branding} Search UI application provides a graphic interface to return results in HTML format and locate them on an interactive globe or map.
-For more details on using this application, go to {branding} Search UI User's Guide.
+===== ${ddf-ui} Application
+
+The ${branding} Search UI application provides a graphic interface to return results and locate them on an interactive globe or map.
```
Comment (vinamartin) [inline distribution/docs/src/main/resources/_contents/integrating-contents.adoc]: This seems redundant to me. You could just use `${ddf-ui}` again here.

Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/integrating-contents.adoc]: Yes. It did get fixed in DDF-1861.

**Pull request merged**